index-test.html

```diff
@@ -54,14 +62,45 @@
               console.log("Rating chosen: ", r.rating);
             };
             $rootScope.regionCode = "WA";
-          },
+            $rootScope.updateDashboardCases = (d) => {
+              $rootScope.statusToFilterOn = d;
+              console.log("Selected cases filter:", d);
+            };
+            // variables for stat card
+            $rootScope.userNum = "20";
+            $rootScope.statusToFilterOn = "cases_pending";
+            $rootScope.isVacationMode = "true";
+            $rootScope.updateVacationMode = () => {
+              $rootScope.isVacationMode = !$rootScope.isVacationMode; 
+            }
+
+            // variables for toggle card
+            $rootScope.message = "You are NOT accepting cases"
+            $rootScope.uibMessage = "Accepting Cases is turned off for your law firm. You will not receive any new cases. If you wish to resume receiving cases, you can click the toggle switch to start receiving cases again at any time."
+            $rootScope.title = "Not accepting cases";
+            $rootScope.vacationModeUpdating = "false"
+            
+            $rootScope.changeMessage = () => {
+              $rootScope.message = "You are accepting cases";
+              $rootScope.uibMessage = "Accepting Cases is turned on for your law firm. You will receive new cases to review as they are booked. If you need a break or are on vacation, you can click the toggle switch to pause receiving cases at any time.";
+              $rootScope.title = "Accepting cases";
+              $rootScope.vacationModeUpdating = "true";
+              
+              setTimeout(() => {
+                $rootScope.vacationModeUpdating = "false";
+                $rootScope.isVacationMode = !$rootScope.isVacationMode;
+                $rootScope.$apply();
+              }, 3000)
+            }
+          }, 
```

Comment:
This is really neat and great way to test the async toggle!

Nitpick, but it would be nice if the message didn't change until the settimeout finished, and it would also be great to be able to flip back and forth and have the message change.

Also, the toggle on the card itself doesn't have the same effect as a the "toggle card" button but it probably should.

Reply:
I had just noticed that after i sent it lol